File: app/src/main/java/org/zeroxlab/momodict/archive/Word.java
package org.zeroxlab.momodict.archive;

public class Word {
    public IdxEntry entry;
    public String data;

    public String toString() {
        StringBuilder sb = new StringBuilder();
        sb.append(entry);
        sb.append("\n");
        sb.append(data);
        return sb.toString();
    }
}

File: app/src/main/java/org/zeroxlab/momodict/archive/Word.kt
package org.zeroxlab.momodict.archive

class Word {
    var entry: IdxEntry? = null
    var data: String? = null

    override fun toString(): String {
        val sb = StringBuilder()
        sb.append(entry)
        sb.append("\n")
        sb.append(data)
        return sb.toString()
    }
}
